oci_python_image/py_image_layer.bzl

```diff
@@ -0,0 +1,61 @@
+"Simple macro to create tar files from a Python binary."
+
+load("//workaround_rules_pkg_153:runfiles.bzl", "runfiles")
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
+
+def py_image_layer(name, binary, root = None, **kwargs):
+    """Creates two tar files `:<name>/app.tar` and `:<name>/node_modules.tar`
```

Comment:
nit: outdated

Reply:
what's the updated docstring? I'm trying to decipher from

```
bazel query --output=label_kind hello_world:all
_write_file rule //hello_world:_tarball_write_tags
platform rule //hello_world:aarch64_linux
py_library rule //hello_world:hello_world
py_binary rule //hello_world:hello_world_bin
filegroup rule //hello_world:hello_world_layer
pkg_tar_impl rule //hello_world:hello_world_layer/app
runfiles rule //hello_world:hello_world_layer/app/runfiles
oci_image rule //hello_world:image
oci_tarball rule //hello_world:tarball
container_structure_test rule //hello_world:test
platform_transition_filegroup rule //hello_world:transitioned_image
platform rule //hello_world:x86_64_linux
```